.changeset/wet-onions-mate.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Maybe let's delete this changeset, and add the description to the other changeset of `modern-planets-fry`

Reply:
@benjdlambert Oh yes, did the change. Have kept `modern-planets-fry`